Add docs in vaccum.xiaomi_miio.markdown with how to clean a specific room

danielkihlgren opened the pull request:
…w to clean a specific room

It specifies how the command send_command can be used for Xiaomi vacuums.

**Description:**


**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

danielkihlgren committed:
Add documentation in vaccum.xiaomi_miio.markdown with instructions how to clean a specific room

It specifies how the command send_command can be used for Xiaomi vacuums.

frenck commented on source/_integrations/vacuum.xiaomi_miio.markdown:
````diff
@@ -179,6 +179,35 @@ automation:
           - 26496
 ```
 
+### Service `vacuum.send_command`
````
This service works with this integration, but is not of this integration (it belongs to the `vacuum` integration) and should not be documented or listed on this page.

I would suggest adding an additional section with examples in this case.

danielkihlgren committed:
Update Xiaomi vacuum docs on how to clean a room

An example is specified under a new section. The service vacuum.send_command is not documented here anymore.

frenck committed:
:pencil2: Tweaks

frenck reviewed (approved):
Thanks, @danielkihlgren!

Awesome addition to our documentation, thanks! 👍
